Source/WebKit/NetworkProcess/EntryPoint/Cocoa/XPCService/NetworkService/Info-OSX.plist

```diff
@@ -36,6 +36,8 @@
 		<true/>
 		<key>RunLoopType</key>
 		<string>NSRunLoop</string>
+                <key>_ProcessType</key>
+                <string>App</string>
```

Comment:
As a follow up we should consider changing this to `Interactive` instead of `App` and rely on our RunningBoard assertions to grant us more CPU time.